lib/build/helpers/TaskUtil.js

```diff
@@ -163,6 +163,16 @@ class TaskUtil {
 		return this._projectBuildContext.getOption(key);
 	}
 
+	/**
+	 * Callback that will be executed once the build is finished
+	 *
+	 * @public
+	 * @callback @ui5/project/build/helpers/TaskUtil~cleanupTaskCallback
+	 * @param {boolean} force Specifies whether the cleanup callback should
```

Comment:
The current implementation doesn't include a default behaviour (to false), so as `executeCleanupTasks` is called with undefined, the callbacks also receive undefined, but this does not fit to the type `{boolean}` in here.